docs/gh-action.rst

```diff
@@ -0,0 +1,136 @@
+==============================
+Using Bowtie in GitHub Actions
+==============================
+
+You can now use bowtie as a GitHub Action. Follow the steps to get started with it in a few simple steps:
```

Comment:
Rather than saying "now", which will go out of date, best to just stick to the present tense, i.e.:

```suggestion
Bowtie can be used from within GitHub Actions.
Follow these steps to get started with it in a few simple steps:
```

Also have a quick look at [this style guide on semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) -- in short, generally put each sentence on its own line in the file here (that will still get combined into paragraphs in the rendered docs).